apps/src/dom.js

```diff
@@ -30,7 +30,10 @@ var addEvent = function(element, eventName, handler) {
   var touchEvent = exports.getTouchEventName(eventName);
   if (touchEvent) {
     element.addEventListener(touchEvent, function(e) {
-      e.preventDefault();  // Stop mouse events.
+      // Stop mouse events and suppress default event handler to prevent
+      // unintentional double-clicking
+      e.preventDefault();
+      element.removeEventListener(eventName, handler);
```

Comment:
we're binding two event listeners here; one to `touchEvent` and one to `eventName`; this is unbinding `eventName` when `touchEvent` is fired; the `touchEvent` will still get fired every time.